parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -271,6 +289,9 @@ MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
 {{if not EnablePodSecurityPolicy}}
     sed -i "s|apparmor_parser|d|g" "/etc/systemd/system/kubelet.service"
 {{end}}
+{{if EnableDataEncryptionAtRestWithExternalKms}}
+    sed -i "s|docker.service|docker.service kms.service|g" "/etc/systemd/system/kubelet.service"
```

Comment:
so we are chaining kubelet on `kms` and then chaining `kms` on `docker`? seems a bit risky. Can we make it depend on `both` instead of chaining?

Shouldn't we escape the . in `docker.service` and `docker.service kms.service`

Reply:
Are you saying we should update kubelet.service as:
`Requires=docker.service kms.service` 
`After=docker.service`
and kms.service as:
`Requires=docker.service` 
`After=network-online.target`